static/docs/command-reference/pipeline/show.md

```diff
@@ -52,6 +53,35 @@ instead of stages.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
+## Paging the output
+
+This command's output is automatically piped to
+[`less`](<https://en.wikipedia.org/wiki/Less_(Unix)>), if available in the
+terminal. (The exact command used is `less --chop-long-lines --clear-screen`.)
+If less is not available (e.g. on Windows), the output is simply printed out.
+
+> It's also possible to
+> [enable Less paging on Windows](/doc/user-guide/running-dvc-on-windows#enabling-paging-with-less).
+
+### Providing a custom pager
+
+It's possible to override the default pager via the `DVC_PAGER` environment
+variable. For example, the following command will replace the default pager with
+[`more`](<https://en.wikipedia.org/wiki/More_(command)>), for a single run:
+
+```bash
+$ DVC_PAGER=more dvc pipeline show --ascii my-pipeline.dvc
+```
+
+For a persistent change, define `DVC_PAGER` in the shell configuration. For
+example in Bash, we could add the following line to `~/.bashrc`:
+
+```bash
+export DVC_PAGER=more
+```
+
+to `~/.bashrc`.
```

Comment:
Please remove these 2 lines.

Reply:
(already mentioned in the revised paragraph above.)

Reply:
good catch and done